tools/triage_stats.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# pylint: disable=invalid-name,bare-except
+
+# Script to output table with the count of triage related tickets, grouped by week for the past 3 weeks
+
+import json
+import tabulate
+import jira_cmd
+from collections import defaultdict
+
+
+class TriageStats:
+    def __init__(self):
+        parser = jira_cmd.build_parser()
+        def my_exit(msg):
+            pass
+        parser.error = my_exit
+        self.args = parser.parse_args()
+        self.args.print_json = True
+        self.args.linked_issues = True
```

Comment:
I don't get it
Why we need to call it with the cli parser? Can't we call a regular function?

Reply:
Since most of the code get the parser struct, we need to build this struct and populate it